experiments/dagger_benchmark.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+set -e
+source experiments/common.env
+
+ENVS="cartpole"
+DATA_DIR=${DATA_DIR:-"data"}
+LOG_ROOT="output/dagger_benchmark/${TIMESTAMP}"
+extra_configs=""
+extra_options=""
+
+SEEDS="0 1 2 3 4"
+
+TEMP=$($GNU_GETOPT -o fT -l fast,mvp,mvp_fast,tmux,pdb,echo,run_name:,log_root:,file_storage:,mvp_seals -- "$@")
+if [[ $? != 0 ]]; then exit 1; fi
+eval set -- "$TEMP"
+
+while true; do
+  case "$1" in
+    # Fast mode (debug)
+    -f | --fast)
+      SEEDS="0"
+      extra_configs+="fast "
+      shift
+      ;;
+    --mvp_seals)  # Table benchmark settings
+      ENVS="seals_cartpole seals_mountain_car half_cheetah"
```

Comment:
You're not using the `half_cheetah` version of seals though? Consider renaming this to just `--mvp`.

Reply:
Ah, saw your TODO in `bc_benchmarks.sh` -- I've fixed this now.

Reply:
Great, thanks!